engineering/methodology.md

```diff
@@ -108,16 +108,14 @@ Demo meetings
 
   + Starting at the end of the second sprint, one every other sprint.
 
-  + Every 2nd week of sprint, on Fridays after the [sprint demo](https://github.com/src-d/guide/blob/master/engineering/methodology.md#Demo).
+  + Every 2nd week of sprint, on Fridays after the [sprint demo](#Demo).
 
 ### Keeping track of things:
 
   + Publicly: a Kanban board on-line.
 
   + Per team: any way team sees fit (github issues, github projects, waffle.io...).
```

Comment:
while we are still here, may be "team sees fit (most teams use github projects)"

Reply:
I won't change this since it should be reflected in the PR on methodology